src/fides/api/models/privacy_experience.py

```diff
@@ -1,94 +1,252 @@
     """
-    Whether the banner should display - not formalized in the db
+    Common fields shared between:
+    - ExperienceConfigTemplate
+    - PrivacyExperienceConfig
+    - PrivacyExperienceConfigHistory
+
+    These are non-translated fields.
     """
 
-    always_enabled = "always_enabled"
-    enabled_where_required = "enabled_where_required"  # If the user's region has at least one opt-in or notice-only notice
-    always_disabled = "always_disabled"
+    allow_language_selection = Column(
+        Boolean, nullable=False, default=True, server_default="t"
+    )
+
+    @declared_attr
+    def component(cls) -> Column:
+        return Column(
+            EnumColumn(ComponentType),
+            nullable=False,
+            index=True,
+        )
+
+    disabled = Column(Boolean, nullable=False, default=True)
+    dismissable = Column(Boolean, nullable=False, default=True, server_default="t")
+    name = Column(String)
+
+
+class ExperienceConfigTemplate(PrivacyExperienceConfigBase, Base):
+    """Table for out-of-the-box Experience Configurations"""
+
+    privacy_notice_keys = Column(
+        ARRAY(String)
+    )  # A list of notice keys which should correspond to a subset of out-of-the-box notice keys
+    regions = Column(
+        ARRAY(EnumColumn(PrivacyNoticeRegion, native_enum=False)),
+    )
+    translations = Column(
+        ARRAY(JSONB)
+    )  # A list of all available out of the box translations
 
 
-class ExperienceConfigBase:
-    """Base schema for PrivacyExperienceConfig."""
+class ExperienceTranslationBase:
+    """Base schema for fields shared between ExperienceTranslation and PrivacyExperienceHistory."""
 
     accept_button_label = Column(String)
     acknowledge_button_label = Column(String)
     banner_description = Column(String)
-    banner_enabled = Column(EnumColumn(BannerEnabled), index=True)
     banner_title = Column(String)
-    component = Column(EnumColumn(ComponentType), nullable=False, index=True)
     description = Column(String)
-    disabled = Column(Boolean, nullable=False, default=False)
     is_default = Column(Boolean, nullable=False, default=False)
+
+    language = Column(
+        EnumColumn(
+            SupportedLanguage,
+            native_enum=False,
+            values_callable=lambda x: [
+                i.value for i in x
+            ],  # allows enum _values_ to be stored rather than name
+        ),
+        nullable=False,
+    )
+
     privacy_policy_link_label = Column(String)
     privacy_policy_url = Column(String)
     privacy_preferences_link_label = Column(String)
     reject_button_label = Column(String)
     save_button_label = Column(String)
     title = Column(String)
-    version = Column(Float, nullable=False, default=1.0)
 
 
-class PrivacyExperienceConfig(ExperienceConfigBase, Base):
-    """Stores common copy to be shared across multiple regions, e.g.
-    banner titles, descriptions, button labels, etc.
+class DeprecatedConfigFields:
```

Comment:
A follow-up commit after we get the data migration in, will remove this class, and other fields marked deprecated.